### ISSUE TITLE
Values are not populated in left panel  in Geoportal Server Catalog 2.6.4 for Organizations, Topic Category and Time Period

### ISSUE THREAD

**Issue by Dineshr25** [Esri/geoportal-server-catalog#382]
Dear ESRI Team,

It would be appreciable if you can assist on below mentioned issue.

Values are not getting populated in left panel  in Geoportal Server Catalog 2.6.4 for Metadata Type:-ArcGIS:
1. Organizations
2. Topic Category
3. Time Period

Thanks & Regards
Dinesh

**Comment by mhogeweg** [comment]
Could you attach an ArcGIS XML you are indexing with these fields populated? That would allow us to compare how you filled the metadata with how the ArcGIS evaluator interprets it.

Also, if can you confirm you can see the ArcGIS records, could you also attach the json representation of the same record as your ArcGIS xml?

**Comment by Dineshr25** [comment]
Thanks for your quick response.

Please find attached bellow required XML and JSON file for analysis further.

[Test.zip](https://github.com/Esri/geoportal-server-catalog/files/6735118/Test.zip)

Regards
Dinesh

**Comment by mhogeweg** [comment]
ok. a quick look explains why you're not seeing any facets populated. The metadata is very sparse. Geoportal will look for these specific elements in your metadata:
```
    G.evalProp(task,item,root,"fileid","mdFileID");
    G.evalProp(task,item,root,"title","dataIdInfo/idCitation/resTitle");
    G.evalProp(task,item,root,"description","dataIdInfo/idAbs");
    G.evalProps(task,item,root,"keywords_s","//TopicCatCd/@value | //keyword");
    G.evalProps(task,item,root,"links_s","//distInfo//linkage");
    G.evalProp(task,item,root,"thumbnail_s","dataIdInfo/graphOver/bgFileName");
```

you can find and update this in the EvaluatorFor_ArcGIS.js that is located in geoportal\WEB-INF\classes\metadata\js.

**Comment by Dineshr25** [comment]
Dear Marten,

Thanks for your quick response.

But, even after updating details mentioned by you for js file, I am facing above mentioned issue . Please suggest further.

Thanks & Regards
Dinesh

**Comment by mhogeweg** [comment]
can you share the updated JSON response?

**Comment by Dineshr25** [comment]
Dear Marten,

Please find attached bellow updated JSON file.

[Test2.zip](https://github.com/Esri/geoportal-server-catalog/files/6735841/Test2.zip)

Thanks & Regards
Dinesh

**Comment by mhogeweg** [comment]
hmm, that didn't have any more metadata than the earlier one...

**Comment by Dineshr25** [comment]
Yes. But, how, we can rectify this. Please suggest.

Thanks & Regards
Dinesh

**Comment by mhogeweg** [comment]
you will have to put more information in the ArcGIS metadata .xml itself. Are you using ArcMap/ArcGIS Pro?

**Comment by Dineshr25** [comment]
Dear Marten,

I have added metadata through Geoportal Server Catalog 2.6.4. I have following steps:

Click on Create Metadata-->New Document-->ArcGIS Metadata.
Then I filled all mandatory fields including url for map service which I have published using ArcMap, Then click on save button.
Then, metadata was saved successfully. But, I am not able to see below mentioned values in left panel in Geoportal Server Catalog 2.6.4.

1. Organizations
2. Topic Category
3. Time Period
